source/edge-server/connect.txt

```diff
@@ -10,4 +10,147 @@ Connect to an Edge Server
+             Edge Server config's ``hostname`` field.
+          
+         * - ``appId``
+           - The App ID of the App Services App you created above.
+
+   .. step:: Adjust the App Transport Security Settings
+
+      For iOS, Apple disables insecure connections by default. You must add
+      an exception for your public-addressable DNS to allow the client app 
+      to connect to it.
+
+      Select your App target, and go to the :guilabel:`Signing & Capabilities`
+      tab. Press the :guilabel:`+ Capability` button, and search for 
+      ``App Transport Security Settings``. Add this to your app.
+
+      Add an exception domain for your public-addressable DNS. This should
+      be just the string domain, similar to: ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.
+
+      This creates an :file:`Info.plist` file in your project. Go to this file
+      to make additional adjustments.
+
+      Change this ``String`` key to a ``Dictionary``. Add two more keys for:
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Key
+           - Type
+           - Value
+
+         * - ``NSIncludesSubdomains``
+           - Boolean
+           - ``YES``
+          
+         * - ``NSExceptionAllowsInsecureHTTPLoads``
+           - Boolean
+           - ``YES``
+
+      This enables your iOS client to connect to the insecure HTTP DNS.
+
+      Your :file:`Info.plist` file should look similar to:
+
+      .. image:: /images/tiered-sync-info-plist.png
+         :alt: Screenshot of an Info.plist file showing the settings above
+
+   .. step:: Adjust the Log Level (Optional)
+
+      You can adjust the log level if you'd like to get additional details 
+      about the status of communication with the Edge Server.
+
+      In the :file:`App` file, you can add an ``.onAppear`` block to set 
+      the log level. Add a log level line in this block to get additional 
+      information:
+
+      .. code-block:: swift
+         :emphasize-lines: 10-12
+
+         var body: some Scene {
+            WindowGroup {
+               ContentView(app: app)
+                  .environmentObject(errorHandler)
+                  .alert(Text("Error"), isPresented: .constant(errorHandler.error != nil)) {
+                     Button("OK", role: .cancel) { errorHandler.error = nil }
+                  } message: {
+                     Text(errorHandler.error?.localizedDescription ?? "")
+                  }
+                  .onAppear {
+                     Logger.shared.level = .trace
+                  }
+            }
+         }
+
+      For more information about setting the log level, refer to 
+      :ref:`swift-logging`.
+
+   .. step:: Run the App
+
+      Now you can build and run the app. When you log in, the app connects 
+      to the Edge Server. When you create new Items, those sync with the 
+      Edge Server, and from there to the Atlas App Services App.
```

Comment:
```suggestion
      Edge Server, and then sync from the Edge Server to the Atlas App Services App.
```

Reply:
I think I can be a bit more accurate so I'm rewording this section a bit.